src/ramses_tx/helpers.py

```diff
@@ -721,8 +721,8 @@ def parse_fan_info(value: HexStr2) -> PayDictT.FAN_INFO:
     if not isinstance(value, str) or len(value) != 2:
         raise ValueError(f"Invalid value: {value}, is not a 2-char hex string")
 
-    # if value == "EF":  # TODO: Not implemented???
-    #     return {SZ_FAN_INFO: None}
+    if value in "1F, EF, FF":  # 1F constant in HRU which has actual fan_info in 31D9
+        return {SZ_FAN_INFO: "", "_unknown_fan_info_flags": []}
```

Comment:
I believe EF means None, or N/A - FF may be similar.

1F is completely different - it means _something_.

Reply:
I have split this, passing along the `0x1F` value and filter on it instead.